docker-run.sh

```diff
@@ -0,0 +1 @@
+sudo docker run --name x1plus -it -v `pwd`:/work x1plus
```

Comment:
should we have users run the prebuilt docker image instead of building it themselves?  then we know they'll get the same thing we have in CI